updater/src/updater.rs

```diff
@@ -245,6 +294,82 @@ impl Updater {
 		})
 	}
 
+	fn release_block_number(&self, from: BlockNumber, release: &ReleaseInfo) -> Option<BlockNumber> {
```

Comment:
Pretty big piece of code, can I get a comment about what it does and when to use it?

Reply:
It has a comment up there in the trait definition:

> /// Fetches the block number when the given release was added, checking the interval [from; latest_block].

I can make it more descriptive if you feel it's necessary.

Reply:
Ah sorry, I wasn't looking back at the trait to compare.